src/Playwright.TestingHarnessTest/Playwright.TestingHarnessTest.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
Will it break those who haven't updated to net8 yet?

Reply:
These are our internal tests, its not affecting users.